### PR TITLE
Handle CORS in postflight

### PR DESCRIPTION
**Pull request by rupertbarton**

### CODE REVIEW

File: api/app.py [hunk 1]
```diff
@@ -21,17 +21,24 @@ def handle_preflight():
         return _build_cors_preflight_response()
 
 
+@app.after_request
+def handle_postflight(response):
```
Comment (sophie-richardson25) [inline api/app.py:25]: Do you not need to jsonify it too? or is it already jsonified at this point

Comment (rupertbarton) [reply api/app.py:25]: No idea, but it seems to work haha

Comment (rupertbarton) [reply api/app.py:25]: Do you wanna give it a try?

Comment (sophie-richardson25) [reply api/app.py:25]: haha it does seem to work, when it definitely didn't before without the jsonify!! ah well

**Review by sophie-richardson25 — APPROVED** [review]

**Pull request merged**